### PR TITLE
Update scalaz version

### PR DESCRIPTION
**Pull request by shinnya**
Hi, @gakuzzzz.

I've used free-scalikejdbc for my commercial products and want to update scalaz version for some reason. But scalaz doesn't provide `FreeC` type anymore from `7.2`. So I fixed the code to make all unit tests green.

I updated the scalaz version to the latest and is it OK for you?

See this commit for more detail:
https://github.com/scalaz/scalaz/commit/783eb0ed7f6aec56d6245e9b2b9bac97aab74d22.

Sincerely.

### CODE REVIEW

**Comment by shinnya** [comment]
I would be happy if you could review in a day or two 😄

File: build.sbt [hunk 1]
```diff
@@ -17,9 +17,9 @@ lazy val core = (project in file("core")).settings(
   resolvers += "Scalaz Bintray Repo" at "https://dl.bintray.com/scalaz/releases",
   libraryDependencies ++=
     ("org.scalikejdbc"         %% "scalikejdbc"                      % scalikejdbcVersion) ::
-    ("org.scalaz"              %% "scalaz-core"                      % "7.1.3") ::
-    ("org.scalaz"              %% "scalaz-concurrent"                % "7.1.3") ::
-    ("org.scalaz.stream"       %% "scalaz-stream"                    % "0.7.1a") ::
+    ("org.scalaz"              %% "scalaz-core"                      % "7.2.6") ::
+    ("org.scalaz"              %% "scalaz-concurrent"                % "7.2.6") ::
+    ("org.scalaz.stream"       %% "scalaz-stream"                    % "0.8.4") ::
```
Comment (xuwei-k) [inline build.sbt]:
``` scala
"org.scalaz.stream"  %% "scalaz-stream" % "0.8.4a"
```

https://oss.sonatype.org/content/repositories/releases/org/scalaz/stream/scalaz-stream_2.11/0.8.4a/scalaz-stream_2.11-0.8.4a.pom
https://oss.sonatype.org/content/repositories/releases/org/scalaz/stream/scalaz-stream_2.11/0.8.4/scalaz-stream_2.11-0.8.4.pom

**Review by xuwei-k — CHANGES_REQUESTED** [review]
inconsistent scalaz-stream version

File: build.sbt (hunk 1 above)
Comment (shinnya) [reply build.sbt]: Thanks. I'll update scalaz-stream to 0.8.4a and scalaz-\* to 7.2.5.

Comment (shinnya) [reply build.sbt]: fixed

**Comment by gakuzzzz** [comment]
@shinnya sorry my late reply.

This PR LGTM!!

I want to merge this. however it conflicts #4.
Would you solve it? 
I'll merge after it.

thanks!

**Comment by shinnya** [comment]
@gakuzzzz Ok. I'll fix soon.

**Comment by gakuzzzz** [comment]
thanks a lot!!

**Comment by shinnya** [comment]
Thank you :smile:

**Comment by shinnya** [comment]
@gakuzzzz BTW, when are you going to publish a new version to Maven Central? I would appriciate it if you could publish as soon as possible.

**Comment by shinnya** [comment]
Oops. I forgot that free-scalikejdbc was not registered on Maven Central. Sorry.

**Comment by gakuzzzz** [comment]
yah, free-scalikejdbc is a Proof of Concept. there are no plans to publish into maven central.

**Pull request merged**